rust-runtime/aws-smithy-runtime/src/client/http/body/minimum_throughput/throughput.rs

```diff
@@ -97,90 +97,264 @@ impl From<(u64, Duration)> for Throughput {
     }
 }
 
-#[derive(Clone)]
-pub(super) struct ThroughputLogs {
-    max_length: usize,
-    inner: VecDeque<(SystemTime, u64)>,
-    bytes_processed: u64,
+/// Cell in a linear grid that represents a small chunk of time.
+#[derive(Copy, Clone, Debug)]
+enum Cell {
```

Comment:
_nitpick_

Should we avoid calling this `Cell` because of the connotations that that has for Rust programmers? Typically `Cell` means some sort of pointer type.

As an alternative, I offer the equally meaningless `Bin`